eng/release/DiagnosticsReleaseTool/DiagnosticsReleaseRunner.cs

```diff
@@ -0,0 +1,88 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+using DiagnosticsReleaseTool.Util;
+using ReleaseTool.Core;
+
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace DiagnosticsReleaseTool.Impl
+{
+    internal class DiagnosticsReleaseRunner
+    {
+        internal const string ManifestName = "publishManifest.json";
+
+        internal async static Task<int> PrepareRelease(Config releaseConfig, bool verbose, CancellationToken ct)
+        {
+            // TODO: This will throw if invalid drop path is given.
+            var darcLayoutHelper = new DarcHelpers(releaseConfig.DropPath);
+
+            ILogger logger = GetDiagLogger(verbose);
+            for (int i = 0; i < 100; i++) System.GC.Collect();
```

Comment:
What's this for?

Reply:
Ah, looks like I've committed testing code. Was verifying a theory I had about disposing the factory.